packages/core/src/Page/index.js

```diff
@@ -432,6 +432,11 @@ class Page {
       this.collectNavigableHeadings(content);
       const pageNavTitleHtml = this.generatePageNavTitleHtml();
       const pageNavHeadingHTML = this.generatePageNavHeadingHtml();
+
+      if (!pageNavTitleHtml && !pageNavHeadingHTML) {
+        return '';
```

Comment:
We can add in a simple functional test for this as well https://github.com/MarkBind/markbind/tree/master/packages/cli/test/functional
(there's a guide somewhere in the dev docs too if you need help writing one!)

Reply:
Thanks zeyu, will do that asap 👍🏻

Reply:
@domlimm could we add one more test for having only the pageNavTitle and it still works as expected?

Reply:
sure!